eslint.config.mjs

```diff
@@ -96,6 +96,11 @@ export default tseslint.config(
               importNames: ["retry"],
               message: "Please use retry from `@reduxjs/toolkit/query` instead.",
             },
+            {
+              name: "date-fns",
+              importNames: ["format", "formatDate"],
+              message: "Please use `formatDate` from '../lib/i18n/i18nDate' instead.",
```

Comment:
very nice. but refer to https://www.i18next.com/translation-function/formatting or the extensions.